### PR TITLE
Added indexes to sql commands used to create session tables

### PR DESCRIPTION
**Pull request by JohnstonCode**
As per https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.4.md#httpfoundation `PdoSessionHandler` now precalculates the expiry timestamp in the lifetime column, so the index will speed up garbage collection

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Christopher, thanks for this proposal!

~I think it's a good idea because, in a real app, this table will be accessed a lot. However, looking at PdoSession class, all `SELECT` and `UPDATE` queries use the `id` column as the only `WHERE` clause. Should we create the index on that column instead? I don't know, so I'm just asking.~

Sorry ... I misread the description of your PR, so everything I said is wrong. Let's forget it 😊 

Before merging I'll wait for others to review this. Thanks!

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by javiereguiluz** [comment]
Thank you Christopher.

**Pull request merged**
